### ISSUE TITLE
"Module has no attribute" for attributes coming from dateutil.parser._parser

### ISSUE THREAD

**Issue by duncancmt** [python/mypy#8878]
Hello! I'm having an issue getting my project which depends on `dateutil` to typecheck. I'm not sure if this is a lack of knowledge of the finer points of python typechecking or whether this is a bug in mypy or in typeshed. Thank you for your help.

Please provide more information to help us understand the issue:

* Are you reporting a bug, or opening a feature request?

A bug

* Please insert below the code you are checking with mypy,
  or a mock-up repro if the source is private. We would appreciate
  if you try to simplify your case to a minimal repro.

`scratch.py`:
```python
import dateutil.parser

print(dateutil.parser.ParserError)
```
```
$ python3 scratch.py 
<class 'dateutil.parser._parser.ParserError'>
```
```
$ python3 -c 'import dateutil ; print(dateutil.__version__)'
2.8.1
```

* What is the actual behavior/output?

```
$ mypy --strict scratch.py 
scratch.py:3: error: Module has no attribute "ParserError"
Found 1 error in 1 file (checked 1 source file)
```

* What is the behavior/output you expect?

No error reported

* What are the versions of mypy and Python you are using?

```
$ mypy --version
mypy 0.770
```
```
$ python3 --version
Python 3.8.2
```
* Do you see the same issue after installing mypy from Git master?

Yes:
```
$ mypy --version
mypy 0.770+dev.05571b17a29ef83708ffaa02756e32e60621f69f
$ mypy --strict scratch.py 
scratch.py:3: error: Module has no attribute "ParserError"
Found 1 error in 1 file (checked 1 source file)
```

* What are the mypy flags you are using? (For example --strict-optional)

`--strict`

**Comment by ethanhs** [comment]
It appears that this is missing from the typeshed stub https://github.com/python/typeshed/blob/master/third_party/2and3/dateutil/parser.pyi, you should open an issue there.
